### PR TITLE
docs: update

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

- update local playground docs including
  - referencing postman environments
  - moving autopeering to own page
- update integration docs re: fees
- add link to OpenAPI specs for webhooks and rates endpoint to integration docs

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [x] Documentation added
- [ ] Make sure that all checks pass
- [ ] Postman collection updated

### CODE REVIEW

File: packages/documentation/src/content/docs/playground/autopeering.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+// OR to start with Postgres db
+pnpm localenv:compose:psql:autopeer
+```
+
+Your local Rafiki instance will be peered automatically with the remote [Rafiki.money](https://rafiki.money) instance.
+The required services will be exposed externally using [tunnelmole](https://www.npmjs.com/package/tunnelmole) package.
+The exposed ports are 3000(open-payments), 3006(auth server), 3002(ILP connector).
+
+To use the [Postman API examples](../overview.md#postman--open-payments-apis) follow the steps:
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/playground/autopeering.md:18]: this one is broken

File: packages/documentation/src/content/docs/playground/autopeering.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Autopeering
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/playground/autopeering.md:2]:
```suggestion
title: Auto-Peering
```
to be the same as side bar

File: packages/documentation/src/content/docs/playground/overview.md [hunk 3]
```diff
@@ -162,24 +137,26 @@ pnpm localenv:compose down --volumes
 
 #### Postman & Open Payments APIs
 
-The Open Payments APIs can be interacted with using the [Postman collection](https://www.postman.com/interledger/workspace/interledger/api/84fc90ca-3153-4865-8b49-b91218e5d574). It is configured to use the default endpoints of the local environment.
+The Open Payments APIs can be interacted with using the [Postman API collection](https://www.postman.com/interledger/workspace/interledger/api/84fc90ca-3153-4865-8b49-b91218e5d574). It requires you to either load the [Local Environment](https://raw.githubusercontent.com/interledger/rafiki/main/postman/environments/Local%20Playground%20Environment.postman_environment.json) or the [Remote Environment](https://raw.githubusercontent.com/interledger/rafiki/main/postman/environments/Remote%20Environment.postman_environment.json).
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/playground/overview.md:140]: should we mention that you probably need to sign in with the postman account and then fork the APIs?

File: packages/documentation/src/content/docs/playground/autopeering.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+pnpm localenv:compose:psql:autopeer
+```
+
+Your local Rafiki instance will be peered automatically with the remote [Rafiki.money](https://rafiki.money) instance.
+The required services will be exposed externally using [tunnelmole](https://www.npmjs.com/package/tunnelmole) package.
+The exposed ports are 3000(open-payments), 3006(auth server), 3002(ILP connector).
+
+To use the [Postman API examples](../overview.md#postman--open-payments-apis) follow the steps:
+
+1. run `docker logs rafiki-cloud-nine-mock-ase-1`
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/playground/autopeering.md:20]: we can also just view the MASE to get these?

File: packages/documentation/src/content/docs/playground/autopeering.md (hunk 1 above)
Comment (mkurapov) [inline packages/documentation/src/content/docs/playground/autopeering.md:18]: Should we mention that you can use open payments client as well?

Comment (sabineschaller) [reply packages/documentation/src/content/docs/playground/autopeering.md:18]: Still? Ok, will fix later. Have the meeting with the hackathon gang now.

**Pull request merged**